### PR TITLE
Add keybinding to open selection in terminal

### PR DESCRIPTION
**Pull request by x4121**

### CODE REVIEW

File: scripts/tmux-extrakto.sh [hunk 1]
```diff
@@ -112,6 +120,11 @@ function capture() {
         capture
       fi
       ;;
+
+    ctrl-e)
+      tmux send-keys -t ! "$editor -- $text"
+      tmux send-keys -t ! 'C-m'
```
Comment (laktak) [inline scripts/tmux-extrakto.sh]: You can do this in one command:
```
      tmux send-keys -t ! "$editor -- $text" 'C-m'
```

File: scripts/tmux-extrakto.sh [hunk 2]
```diff
@@ -36,10 +36,18 @@ if [[ "$open_tool" == "auto" ]]; then
...
+  editor="vi"
+else
+  editor="$EDITOR"
+fi
+
 function capture() {
 
   header="tab=insert, enter=copy"
   if [ -n "$open_tool" ]; then header="$header, ctrl-o=open"; fi
+  header="$header, ctrl-e=open in $editor"
```
Comment (laktak) [inline scripts/tmux-extrakto.sh]: I thik `ctrl-e=edit` should be sufficient (also for example my `$EDITOR` command includes parameters and this looks a bit strange)

**Comment by laktak** [comment]
Works great but could you look at the two comments? thanks!

File: scripts/tmux-extrakto.sh (hunk 1 above)
Comment (x4121) [reply scripts/tmux-extrakto.sh]: Oh nice, didn't know that :slightly_smiling_face:

**Comment by laktak** [comment]
thanks!

**Pull request merged**